File: Small Programs/SqrtExpression.cs
/* [Peter Giblin] Primes and Programming, Computing exercise 4.5 (a) */

using System;

class SqrtExpression
{
    static void Main()
    {
        for (int n = 1; n <= 100; ++n) {
            int tmp = (int)Math.Floor(Math.Sqrt(n)) + 1;
            int d = tmp * tmp - n;
            Console.WriteLine("{0,4}  {1,4}", n, d);
        }
    }
}

File: Small Programs/SqrtExpression.java
/* [Peter Giblin] Primes and Programming, Computing exercise 4.5 (a) */

import static java.lang.Math.*;

public class SqrtExpression
{
    public static void main(String[] args)
    {
        for (int n = 1; n <= 100; ++n) {
            int tmp = (int)floor(sqrt(n)) + 1;
            int d = tmp * tmp - n;
            System.out.printf("%4d  %4d%n", n, d);
        }
    }
}
